### PR TITLE
Refactored `Phalcon\Commands\Command` slightly

### PR DESCRIPTION
**Pull request by SidRoberts**
```php
if (strpos($parameter, "=") !== false) {
    // ...
} else {
    if (strpos($parameter, "=") !== false) {
        // ...
    } else {
        // ...
    }
}
```

### CODE REVIEW

File: scripts/Phalcon/Commands/Command.php [hunk 1]
```diff
@@ -416,9 +384,7 @@ public function getOption($option, $filters = null, $defaultValue = null)
             foreach ($option as $optionItem) {
                 if (isset($this->_parameters[$optionItem])) {
                     if ($filters !== null) {
-                        $filter = new Filter();
-
-                        return $filter->sanitize($this->_parameters[$optionItem], $filters);
+                        return $this->filter($this->_parameters[$optionItem], $filters);
```
Comment (sergeyklay) [inline scripts/Phalcon/Commands/Command.php]: Hmm.. Where `$this->filter` initialized?

File: scripts/Phalcon/Commands/Command.php [hunk 2]
```diff
@@ -134,38 +134,18 @@ public function getScript()
     protected function getConfig($path)
     {
         foreach (array('app/config/', 'config/') as $configPath) {
-            if (file_exists($path . $configPath. "config.ini")) {
-                return new IniConfig($path . $configPath. "/config.ini");
-            } elseif (file_exists($path . $configPath. "/config.php")) {
-                $config = include($path . $configPath. "/config.php");
-                if (is_array($config)) {
-                    $config = new Config($config);
+            foreach (array('ini', 'php', 'json', 'yaml') as $extension) {
```
Comment (sergeyklay) [inline scripts/Phalcon/Commands/Command.php]: I think in this branch we can use short syntax

File: scripts/Phalcon/Commands/Command.php (hunk 1 above)
Comment (SidRoberts) [inline scripts/Phalcon/Commands/Command.php]: https://github.com/SidRoberts/phalcon-devtools/blob/patch-1/scripts/Phalcon/Commands/Command.php#L437

File: scripts/Phalcon/Commands/Command.php (hunk 2 above)
Comment (SidRoberts) [inline scripts/Phalcon/Commands/Command.php]: See f594ec6.

File: scripts/Phalcon/Commands/Command.php (hunk 1 above)
Comment (sergeyklay) [inline scripts/Phalcon/Commands/Command.php]: Oh :+1:

**Comment by sergeyklay** [comment]
This is ready to merge?

**Comment by SidRoberts** [comment]
@sergeyklay Yeah, it is.

**Comment by sergeyklay** [comment]
Thanks

**Comment by sergeyklay** [comment]
This branch will be released immediately after Phalcon 2.1.0 release. We can use here capabilities and features of the Phalcon 2.1.0

**Pull request merged**